defaults/main.yml

```diff
@@ -6,6 +6,7 @@ thinlinc_accept_eula: "no"
 thinlinc_version: "4.12.1"
 thinlinc_build: "6733"
 thinlinc_server_bundle: "tl-4.12.1-server.zip"
+thinlinc_pack: "{{ thinlinc_version }}-{{ thinlinc_build }}"
```

Comment:
Sorry for being unclear, but the goal was one commit per type of change. Not per file. So all of these would be in the same commit.

That said, I think these are just making it difficult to read. So I think we should disable this lint rule, at least for the constants that define the file names.